### ISSUE TITLE
CRE or CRM

### ISSUE THREAD

**Issue by DimmestP** [DimmestP/chimera_project_manuscript#80]
Cis-regulatory element (CRE) and cis-regulatory module (CRM) are apparently both terms already coined for referring to different sized regulatory regions. Although this is typically  Currently the phrase cis-regulatory element module is used in the intro which is a little confusing!

**Comment by DimmestP** [comment]
My current go to is to say CREs for terminator/promoter and short CREs for motifs

**Comment by ewallace** [comment]
I like this approach. It also reduces the number of abbreviations we use.

**Comment by ewallace** [comment]
We discussed this 20th April, with some comments from @daneckaw. We need to go through and check when we use "short CREs" and when we use "motif".

We could
* use "short CRE" almost exclusively, if this works?
* use "motif" mostly, as long as we carefully define our meaning as "motif" as shorthand for our specific set of short cis-regulatory elements

**Comment by ewallace** [comment]
Abstract edits in #135 are pushing towards use "motif" mostly. That still needs us to carefully define our meaning of "motif" as shorthand for our specific set of short cis-regulatory elements.

**Comment by ewallace** [comment]
Discussed with @j-aux 3 June 2021. We agreed that "short CRE" is not in general use and so is not helpful, so we should remove it. However, "motif" is fine and works as long as we define what *we* mean by it carefully. We discussed whether "CRE" was useful as opposed to more precisely promoters and terminators. We argued that "CRE" is an overarching term that should be useful, even though we are using it mostly to refer to promoters and terminators. Let us define it clearly.
